{{ cookiecutter.repo_name }}/.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: drtools/drtools:latest
```

Comment:
Why this image? I've seen that in is24 we're using `docker:17.05.0-ce-git`. What's the advantage of using our image here?

Reply:
The container spawned from this image will be the controller during integration test. So it needs drtools and your project code to work correctly. This avoids having to install drtools in a complicated way.